.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,27 @@
+name: Publish releases to PyPI
+
+on:
+  release:
+    types: [published, prereleased]
+
+jobs:
+  build-and-publish:
+    name: Builds and publishes releases to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2.3.1
+        with:
+          python-version: 3.8
+      - name: Install wheel
+        run: >-
+          pip install wheel
+      - name: Build
+        run: >-
+          python3 setup.py sdist bdist_wheel
+      - name: Publish release to PyPI
+        uses: pypa/gh-action-pypi-publish@main
```

```suggestion
        uses: pypa/gh-action-pypi-publish@v1.4.2
```

Let's pin this this action to a version:
https://github.com/pypa/gh-action-pypi-publish/releases/tag/v1.4.2